src/sctp_transport/mod.rs

```diff
@@ -64,7 +64,9 @@ pub struct RTCSctpTransport {
     pub(crate) dtls_transport: Arc<RTCDtlsTransport>,
 
     // State represents the current state of the SCTP transport.
-    state: AtomicU8, //SCTPTransportState,
+    
+    // bugfix@Udp connection not close (reopen #174) #195
```

Comment:
I don't think we need these `bugfix@udp...` comments, it will be clear from the git history what happened.

Reply:
comment removed

Reply:
comment removed